pkg/model/config/config.go

```diff
@@ -113,3 +115,9 @@ func (r GVK) isEqualTo(other GVK) bool {
 		r.Version == other.Version &&
 		r.Kind == other.Kind
 }
+
+// GetV1UnsupportedError returns the error to inform that V1 is no longer supported
+func (config Config) GetV1UnsupportedError() error {
+	return fmt.Errorf(noticeColor, "The v1 projects are deprecated since Feb 1, 2020.\n"+
+		"See how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
+}
```

Comment:
I don't like this being part of this struct:

1. The instance is not used for anything.
2. We could technically call this method from a config that is v2.

```suggestion
// V1UnsupportedError informs that V1 is no longer supported
var V1UnsupportedError= fmt.Errorf(noticeColor, "The v1 projects are deprecated since Feb 1, 2020.\n"+
		"See how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
```

Additionally I don't know if this should be part of `pkg/model/config` or `internal/config` package. I feel like this is not part of the public and thus should be moved to `internal/config`.

Reply:
The internal cannot be used in pkg so it cannot be part of the internal/config.
Unfortunately, your suggestions here will force we import the pkg. I would like to not move in this way if you do not mind.

Reply:
The `internal/config` can be imported from inside `pkg` subpackages, thats not an issue. There are definetely some examples: https://github.com/kubernetes-sigs/kubebuilder/blob/9552d129f7d12cc6c7072a1d323ed9f4b461d550/pkg/scaffold/api.go#L22

Reply:
done as suggested.